pcdsdevices/pseudopos.py

```diff
@@ -24,24 +29,31 @@ class Parallel(SyncAxesBase):
 
     Like all ``PseudoPositioner`` classes, any subclass of ``PositionerBase``
     will be included in the synchronized move.
-
-
-    Parameters
-    ----------
-    position_mode: ``func``, optional
-        The function to apply to the list of current positions to determine
-        the combined position. By default, this is the minimum of all axis
-        positions.
     """
     pseudo = Cpt(PseudoSingle)
 
-    _default_mode = min
-
-    def __init__(self, *args, position_mode=None, **kwargs):
+    def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self._mode = position_mode or self._default_mode
         self._offsets = {}
 
+    def calc_combined(self, real_position):
+        """
+        Calculate the combined pseudo position.
+
+        By default, this is just the position of our first axis.
+
+        Parameters
+        ----------
+        real_position: `namedtuple`
```

Comment:
Nice! Like the `namedtuple`

Reply:
This is inherited from the PseudoMotor schema